tasks/main.yml

```
@@ -1,36 +1,45 @@
 ---
 # tasks file for mongo
 - name: Get mongodb-org GPG key
-  apt_key:
-    keyserver: keyserver.ubuntu.com
+  ansible.builtin.apt_key:
     id: 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
+    keyserver: keyserver.ubuntu.com
+
+- name: Install prerequisites so apt can use a repo over HTTPS
+  ansible.builtin.package:
+    name:
+      - apt-transport-https
+      - ca-certificates
+      - curl
+      - gnupg2
+      - software-properties-common
 
 - name: Copy sources list for mongodb-org (official MongoDB package repo)
-  copy:
-    src: mongodb-org-3.6.list
+  ansible.builtin.copy:
     dest: /etc/apt/sources.list.d
     mode: 0644
+    src: mongodb-org-3.6.list
 
 - name: Update the cache with the mongodb-org goodness
-  package:
+  ansible.builtin.package:
     update_cache: yes
 
 - name: Install mongo
-  package:
+  ansible.builtin.package:
     name: mongodb-org=3.6.8
```

Should we just bump this to the latest point release?

```suggestion
    name: mongodb-org=3.6.23
```